core/templates/pages/admin-page/admin-page.directive.ts

```diff
@@ -91,12 +91,6 @@ angular.module('oppia').directive('adminPage', [
               AdminRouterService.showTab($location.path().replace('/', '#'));
             });
             ctrl.userEmail = '';
-            AdminDataService.getDataAsync().then(function(response) {
-              ctrl.userEmail = response.user_email;
```

Comment:
Why are we removing this value? (As per the issue description the response doesn't have user_email filed but I'm not sure whether we have to fix this or remove everything.)

Reply:
I thought that the user Email is not being used in the template and there is no use of the async call. I've imported `UserBackendApiService` to get the userEmail here.

Reply:
@FareesHussain, I'm, too not sure whether we need the user Email, can you please confirm whether it's needed or not? (It's okay to use UserBackendApiService if we really need the userEmail field) Let me know if you need any help.

Reply:
@DubeySandeep It is being used in the navbar here  
https://github.com/oppia/oppia/blob/46f2318e78dc5dfeb299b89097fad9f058cc9f9b/core/templates/pages/admin-page/admin-page.directive.html#L1

But I'm not sure why do we need it here

Reply:
@DubeySandeep Here it is only used in the icon on the top right (I don't find any other uses of it )

![image](https://user-images.githubusercontent.com/54740946/107161729-ae143b00-69c4-11eb-9f0c-25771f3d0432.png)

Reply:
I agree with @FareesHussain that this can be removed, as it is needed only as a placeholder under the profile picture.

Reply:
Thanks for your thoughts @vojtechjelinek!

@FareesHussain Can you please remove this and update the PR?

Reply:
Done. Please check